Userland/Libraries/LibWeb/HTML/DecodedImageData.h

```diff
@@ -15,6 +15,8 @@ namespace Web::HTML {
 
 // https://html.spec.whatwg.org/multipage/images.html#img-req-data
 class DecodedImageData : public JS::Cell {
+    JS_CELL(DecodedImageData, JS::Object);
```

Comment:
This `JS_CELL` call is incorrect! The second argument should be `JS::Cell` since that is the base class of `DecodedImageData` :)